lib/jwt_signed_request.rb

```diff
@@ -77,7 +77,11 @@ def self.request_body(request:)
   private_class_method :request_body
 
   def self.verified_headers?(request:, claims:)
-    parsed_headers = JSON.parse(claims['headers'])
+    parsed_headers = begin
+      JSON.parse(claims['headers'])
+    rescue JSON::ParserError
+      {}
```

Comment:
I would really like to understand what a _valid_ use case for when you expect there not to be any headers in the claim would look like. If there isn't one, I'm somewhat skeptical to the idea of silently swallowing this error.